meshtastic/mesh_interface.py

```diff
@@ -136,6 +142,33 @@ def __exit__(self, exc_type, exc_value, traceback):
             logging.error(f"Traceback: {traceback}")
         self.close()
 
+    @staticmethod
+    def _printLogLine(line, interface):
+        """Print a line of log output."""
+        if interface.debugOut == sys.stdout:
+            # this isn't quite correct (could cause false positives), but currently our formatting differs between different log representations
```

Comment:
I think we can fix this by passing the whole LogRecord down to here and then looking at the `log_record.DEBUG` etc. as the BLE code was doing -- I'm happy to change that around but was wondering if there's a reason you didn't.

Reply:
yeah - I did this as a temporary hack kinda expecting that LogRecords are coming and the location for coloring will be moving. (I was just merging in recent changes from master and trying to keep em working)